src/PhpSpreadsheet/CachedObjectStorageFactory.php

```diff
@@ -136,7 +136,7 @@ public static function getCacheStorageClass()
      **/
     public static function getAllCacheStorageMethods()
     {
-        return self::$storageMethods;
+        return array_keys(self::$storageMethods);
```

Comment:
This returns the same array as expected (no change of "public method behaviour").